Sources/protoc-gen-swiftgrpc/Generator-Client.swift

```diff
@@ -175,6 +175,7 @@ extension Generator {
         }
         if asynchronousCode {
           println("/// Asynchronous. Unary.")
+	  println("@discardableResult")
```

Comment:
nit: whitespace?

Reply:
Good catch. I think I lost `set noindent` when swapping machines and the tap was only visible in the diff but not the implementation.

PTAL